app/controllers/browse_controller.rb

```diff
@@ -6,8 +6,7 @@ def index
     setup_content_item_and_navigation_helpers(page)
 
     render :index, locals: {
-      page: page,
-      ab_variant: education_ab_test.requested_variant
+      page: page
```

Comment:
In Ruby it's ok to leave in the trailing comma. This helps keep diffs a little smaller. No need to change this now though.